cfme/tests/cloud/test_cloud_manual.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+# pylint: skip-file
+"""Manual tests"""
+
+import pytest
+
+from cfme import test_requirements
+
+pytestmark = [pytest.mark.ignore_stream("5.9", "5.10", "upstream")]
```

Comment:
**Required** Which version is this test valid for?

Reply:
+1 - This won't get collected against anything with all streams ignored. Please see comments below first, as we may be able to remove this module and tests since they're not valid or covered by automation already.